src/Twee/School/Go to school toilet.twee

```diff
@@ -15,7 +15,7 @@
 	<<else>>\
 		You slowly drop to your knees and reach out with a trembling hand for the zipper on the leader's trousers. He eagerly helps you get his pants down, and you slowly pull his underwear down, letting his thick cock pop free.
 	<<endif>>\
-	@@.hero;Thick, but not that huge.@@ With relief you gaze at it for a moment, transfixed, wondering how it will taste. Your cheeks blush a bit at that last thought. @@.hero;I shouldn't find it attractive.<<if not $kink.futa>> I'm not a faggot, I like girls.<<endif>>@@ Your eyes start to get a little watery, but there's no compassion from the bullies. The other two simply turn their backs to you with the excuse of monitoring whether someone is about to enter the bathroom,\
+	@@.hero;Thick, but not that huge.@@ With relief you gaze at it for a moment, transfixed, wondering how it will taste. Your cheeks blush a bit at that last thought. @@.hero;I shouldn't find it attractive.<<if not $kink.futa>> <<if $kinkAllow.cursing>>I'm not a faggot, <</if>>I like girls.<<endif>>@@ Your eyes start to get a little watery, but there's no compassion from the bullies. The other two simply turn their backs to you with the excuse of monitoring whether someone is about to enter the bathroom,\
```

Comment:
```suggestion
	@@.hero;Thick, but not that huge.@@ With relief you gaze at it for a moment, transfixed, wondering how it will taste. Your cheeks blush a bit at that last thought. @@.hero;I shouldn't find it attractive.<<if not $kink.futa>> <<if $kinkAllow.curse>>I'm not a faggot, <</if>>I like girls.<<endif>>@@ Your eyes start to get a little watery, but there's no compassion from the bullies. The other two simply turn their backs to you with the excuse of monitoring whether someone is about to enter the bathroom,\
```